dashboard/test/controllers/api/v1/pd/workshops_controller_test.rb

```diff
@@ -363,7 +363,7 @@ def workshop_params
     {
       location_address: 'Seattle, WA',
       workshop_type: Pd::Workshop::TYPE_PUBLIC,
-      course: Pd::Workshop::COURSE_CSP,
```

Comment:
It could also just as easily be CSP and a subject. I just changed it to CSF because it didn't matter which and it was less typing ;) The change is that CSP now requires a subject.